tensorflow/lite/micro/tools/make/helper_functions.inc

```diff
@@ -398,3 +398,9 @@ $(word 3, $(subst !, ,$(1))):
 	tensorflow/lite/micro/tools/make/download_and_extract.sh $(subst !, ,$(1))
 THIRD_PARTY_TARGETS += $(word 3, $(subst !, ,$(1)))
 endef
+
```

Comment:
This is a bit hairy... I'll leave it up to Pete, but it might make more sense to have a wildcard for each sub-directory you need (I count 7) in the one place you need this.